presto-tests/src/main/java/com/facebook/presto/tests/ResultsSession.java

```diff
@@ -31,6 +33,8 @@ default void setUpdateCount(long count)
         throw new UnsupportedOperationException();
     }
 
+    default void setWarnings(List<PrestoWarning> warnings) {}
```

Comment:
Silent defaults will make this code hard to reason about at compile time *and* run time.

Reply:
The alternative would be to alter the Kafka and Redis testing ResultSession. I will do that instead. Thanks! Adding that to notes (i.e. avoid silent defaults).